release/models/policy/openconfig-routing-policy.yang

```diff
@@ -54,7 +54,13 @@ module openconfig-routing-policy {
     corresponding action statement is executed.  If the action
     statement has either accept-route or reject-route actions, policy
     evaluation of the current policy definition stops, and no further
-    policy definitions in the chain are evaluated.
+    policy definitions in the chain are evaluated.  If the action
```

Comment:
```suggestion
    policy definitions in the chain are evaluated.  If the actions
```

Reply:
I feel that action (singular) is appropriate as only one action can be configured.